common/logger/src/security.rs

```diff
@@ -18,9 +18,9 @@ use crate::StructuredLogEntry;
 
 /// helper function to create a security log
 pub fn security_log(name: &'static str) -> StructuredLogEntry {
-    StructuredLogEntry::new_named(&name)
-        // set the category to security
-        .security()
+    StructuredLogEntry::new_named("security", &name)
```

Comment:
I was wondering if it wouldn't be better to have a file listing all the categories as constants

Reply:
Yeah, that's I think a next step.  It's only being used in 2 places right now, so we can easily add it / centralize it later.